src/cryptoadvance/specter/hwi_rpc.py

```diff
@@ -235,19 +235,18 @@ def extract_xpub(
             # Client will be configured for testnet if our Specter instance is
             #   currently connected to testnet. This will prevent us from
             #   getting mainnet xpubs unless we set is_testnet here:
-            if not chain or chain == "None":
-                try:
-                    client.chain = (
-                        Chain.TEST
-                        if derivation.split("/")[2].startswith("1")
```

Comment:
Wouldn't it be better to be more specific here, setting the chain to main if derivation.split("/") startswith("0"), to test if startswith("1") and raising some error if sth. else (could also be by mistake) is provided?